City-of-Helsinki/drupal-infofinland
# UHF-10618

# [UHF-10618](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10618)

Fixed the node saving certain nodes.

## What was done
Updated owner of the broken nodes to anon user.
(Also removed the gdpr module code which was disabled ages ago)

## How to reproduce
- Make fresh without the fix
- Go edit one of the broken nodes (check list from code)
  - add a text paragraph, add text to it and save
- Operation will fail on error 500.
- Saving any **other than** broken node should work ok.

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-10618`
  * `make fresh`
* Run `make drush-cr`

## How to test
- Log in as admin and add a text paragraph to a broken node, add text to it and save the page
- You should be able to save without errors.

Also check that the gdpr module waa actually removed from core.extension before this PR


[UHF-10618]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-10618?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
